src/main.cpp

```diff
@@ -17,7 +17,9 @@
 
 #include "MemorySettingsRepository.h"
 
-
+#ifndef _WIN32
+	#include <unistd.h>
```

Comment:
I think this should be in [Globals.h#L207](https://github.com/mc-server/MCServer/blob/master/src/Globals.h#L207) instead